extern/CMakeLists.txt

```diff
@@ -16,14 +16,33 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Android")
         FetchContent_Declare(
             SDL2
 	    GIT_REPOSITORY https://github.com/libsdl-org/SDL.git
-	    GIT_TAG release-2.28.1 
+	    GIT_TAG release-2.28.1
     	)
         message("SDL2 not found. Downloading now...")
         FetchContent_MakeAvailable(SDL2)
         message("SDL2 downloaded to " ${FETCHCONTENT_BASE_DIR}/sdl2-src)
     endif()
 endif()
 
+################################################################################
+# Find/download SDL2 for IOS
+################################################################################
+if (CMAKE_SYSTEM_NAME STREQUAL "iOS")
+    find_package(SDL2 QUIET)
+    if (NOT ${SDL2_FOUND})
+    	include(FetchContent)
+        FetchContent_Declare(
+            SDL2
+            GIT_REPOSITORY https://github.com/libsdl-org/SDL.git
+            GIT_TAG release-2.28.1
+    	)
+        message("SDL2 not found. Downloading now...")
+        FetchContent_MakeAvailable(SDL2)
+        message("SDL2 downloaded to " ${FETCHCONTENT_BASE_DIR}/sdl2-src)
+    endif()
+endif()
```

Comment:
this looks identical to the android one, why not just make the if for that include ios?